nemo/collections/nlp/models/language_modeling/megatron_base_model.py

```diff
@@ -508,16 +508,15 @@ def _get_total_params_across_model_parallel_groups_gpt_bert(self, model):
             num_parameters_on_device = sum(
                 [sum([p.nelement() for p in model_module.parameters()]) for model_module in model]
             )
-            if parallel_state.get_pipeline_model_parallel_world_size() > 1 and parallel_state.is_pipeline_last_stage(
+            if parallel_state.get_pipeline_model_parallel_world_size() > 1 and parallel_state.is_pipeline_first_stage(
```

Comment:
Should this be changed?

Reply:
Yeah because word embeddings are no longer present in the last pipeline stage if you untie embeddings and output weights. But they should always be present in the first pipeline stage.

Reply:
Same for the comment below.